src/EnergyPlus/SteamCoils.cc

```diff
@@ -160,6 +160,12 @@ namespace SteamCoils {
     // MODULE SUBROUTINES:
 
     // Functions
+    void clear_state()
+    {
+        NumSteamCoils = 0;
+        GetSteamCoilsInputFlag = true;
+        SteamCoil.deallocate();
+    }
```

Comment:
Just so you know, the entire struct should be reset on clear_state. @Myoldmopar @mjwitte is this correct?

    // Default Constructor
    OAEquipList()
            : ComponentType_Num(0), ComponentIndex(0), CoilAirInletNode(0), CoilAirOutletNode(0), CoilWaterInletNode(0), CoilWaterOutletNode(0),
      CoilPlantTypeOfNum(0), LoopNum(0), LoopSideNum(0), BranchNum(0), CompNum(0), FluidIndex(0), MaxVolWaterFlow(0.0), MaxWaterMassFlow(0.0),
       MinVolWaterFlow(0.0), MinWaterMassFlow(0.0)